### PR TITLE
Fixes #28556 - Parse Oracle Linux facts properly

### PR DESCRIPTION
**Pull request by jturel**
Fixing per issue reported here: https://community.theforeman.org/t/task-actions-update-results-in-error/16665/4

### CODE REVIEW

**Comment by jturel** [comment]
@jlsherrill any concerns with this? If so I'd like to see if the reporter is willing to test it

**Comment by jlsherrill** [comment]
Looks good to me.  I'd be fine ACKin'g based on user testing

**Comment by pa-jberanek** [comment]
I stumbled across this and fixed it myself, for reference:

https://github.com/Katello/katello/pull/8501

File: app/services/katello/candlepin/consumer.rb [hunk 1]
```diff
@@ -152,6 +152,10 @@ def self.distribution_to_puppet_os(name)
           'Debian'
         elsif name =~ /ubuntu/
           'Ubuntu'
+        elsif name =~ /oracle/
+          'Oracle'
```
Comment (jlsherrill) [inline app/services/katello/candlepin/consumer.rb]: I noticed that the other PR uses 'OracleLinux' and yours uses 'Oracle', what is the proper puppet OS name?

Comment (jturel) [reply app/services/katello/candlepin/consumer.rb]: Looks like OracleLinux is the correct one

https://github.com/puppetlabs/facter/blob/79e5c272cea78dba2fbfac668335529630f79716/lib/inc/facter/facts/os.hpp#L55

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
This has been tested & verified by the reporter!

**Pull request merged**